service/history/workflow/mutable_state_impl.go

```diff
@@ -4558,7 +4559,10 @@ func (ms *MutableStateImpl) ApplyWorkflowExecutionTerminatedEvent(
 	ms.ClearStickyTaskQueue()
 	ms.writeEventToCache(event)
 
-	return ms.processCloseCallbacks()
+	if currentState != enumsspb.WORKFLOW_EXECUTION_STATE_ZOMBIE {
```

Comment:
Is this good enough? 
I mean this will prevent the current cluster from generating callback tasks. But the termination event will eventually be replicated (in event-based replication) to another cluster, then in that cluster callback tasks will still be generated (and processed if failover happens)?